docs/docs/system-administration/system-limits.md

```diff
@@ -38,3 +30,29 @@ Please contact us if you want to override the current system limits.
 
 You can modify the system limits on a per-Project basis. These limits are defined in the database against the Project.
 The easiest way to modify them is with the [Django admin](/deployment/configuration/django-admin.md) interface.
+
+## Traffic Limits
+
+### SDK Limits
+
+Requests made by our SDKs are _not_ rate limited, by design; we can't predict what sort of profile your traffic will
```

Comment:
I can work on rewording this, but requests are technically rate limited. The limit is per month/billing period and not a smaller unit of time like seconds or minutes, so I would leave this out to avoid confusion.

Reply:
Hmm yeah -  I meant "not rate limited in the short term" sort of thing...

Reply:
I think people understand rate limiting to be sort of "near real time" - going over a monthly limit I wouldnt regard as "rate limiting"

Reply:
Yes, I understood what you meant here. My point is that it's not necessary to mention that there are no short-term limits if customers don't need to do anything about that, which also helps prevent confusion between short-term and long-term limits.

Reply:
For free plans we do rate limit after a warning and grace period, so this should be added to this section.

Reply:
Oh I see we've added it below. Could be still better to mention up here.